### ISSUE TITLE
IronNet app available is not the latest version

### ISSUE THREAD

**Issue by marcus-defazio** [splunk-soar-connectors/ironnet#1]
**Name of the app**
IronNet

**Describe the bug**
v1.0.1 does not have the latest changes. The app was updated on Aug 4th 2020 but the new version was never released.

**To Reproduce**
1. Download the IronNet app v1.0.1 from Splunkbase.
2. Extract the contents of the app.
3. Compare the contents of the app to the current version in this repo. It is outdated. 

**Expected behavior**
v1.0.2 is released with the latest code.

**Screenshots**
N/A

**Splunk SOAR Version (please complete the following information):**
 - OVA
 - App Version v1.0.1

**Additional context**
I see the version was bumped to v1.0.2 recently but that version is not available for download.

**Comment by mattsayar-splunk** [comment]
@mjiyani-crest to address the [question](https://github.com/phantomcyber/phantom-apps/pull/248#issuecomment-738870841) you posed on the old repo's PR, I believe the swagger_client wheel is generated via [Swagger's Codegen](https://github.com/swagger-api/swagger-codegen) which uses the Apache 2.0 license.

**Comment by marcus-defazio** [comment]
@mattsayar-splunk and @mjiyani-crest, is there anything left for us to do? Could you release the latest version on splunkbase now?

**Comment by mjiyani-crest** [comment]
> @mattsayar-splunk and @mjiyani-crest, is there anything left for us to do? Could you release the latest version on splunkbase now?

No, we are good for now. We will start our work on this app and release it soon on Splunkbase.

Thanks,
Mit

**Comment by arathore-crest** [comment]
@marcus-defazio, the app currently supports only Python 2 version. Please convert this application to python version 3 if you have valid asset credentials with you, and raise a new PR accordingly with the latest changes. We strongly suggest using Python 3 since Python 2 support will be dropped in the near future.

Thanks!!!

Please refer to [Convert apps from Python 2 to Python 3](https://docs.splunk.com/Documentation/SOARonprem/latest/DevelopApps/Python3).

CC: @mjiyani-crest

**Comment by mattsayar-splunk** [comment]
@marcus-defazio do you plan to convert to Py3?

**Comment by marcus-defazio** [comment]
Yes we do. We are working on it right now and will open a PR in the next 2-3 weeks.

**Comment by marcus-defazio** [comment]
This PR contains the updates: https://github.com/splunk-soar-connectors/ironnet/pull/2
